app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,48 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+  def Show
+
+  end
+  def new
+    @post = Post.find_by_id(params[:post_id])
+  end
+
+  def create
+
+    p "here is there"
+    @post = Post.find(params[:post_id])
+    @comment = @post.comments.create(comment_params) do |comment|
+      comment.user_id=current_user[:id]
+    end
```

Comment:
Learn the best between `find` and `find_by` user what suit best our case.

Reply:
Done